css/stylesheet.css

```diff
@@ -0,0 +1,108 @@
+*{
+    box-sizing:border-box;
+}
+
+body {
+    background-color :#f2f2f2;
+    font-size: 62.5%;
+    font-family: HKGrotesk, sans-serif;
+    height: 100vh;
+    width: 100vw;
+    position: relative;
+}
+
+.hometop {
+    background-color: #F8D5E2;
+    height: 50vh;
+    width: 99vw;
+    padding: 10px 10px 0 10px;
+    margin: 5px 5px 0 5px;
+    position:relative;
+}
+
+nav {
+    width: 98%;
+}
+
+.mypicture {
+    width: 100px;
+    height: 100px;
+    border: 2px solid #adddef;
+    border-radius: 50%;
+    margin: 0;
+    padding: 0;
+}
+
+nav a {
+    text-decoration: none;
+    font-size: 2vh;
```

Comment:
why is this a vh value? that means it will shrink with the screen, is that what you want?

Reply:
Done it on purpose so my text is responsive